### PR TITLE
HODS-404: Updated handler pattern files 

### PR DESCRIPTION
**Pull request by parkeradam**
and updated all areas used accordingly.TEsts updated also.

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -151,42 +151,50 @@ api-config {
...
             title = "Get the organisation's Corporation Tax details"
             endpoint = "/organisations/details/corporation-tax?matchId=<matchId>"
-            fields {
-                A = "taxpayerStartDate",
-                B = "accountingPeriods/apStartDate",
-                C = "accountingPeriods/apEndDate",
-                D = "accountingPeriods/turnover",
-                E = "taxSolvencyStatus"
-            }
+            fields = [A,B,C,D,E]
```
Comment (owen-opencast) [inline conf/application.conf:162]: Add quotes, for consistency

File: conf/application.conf [hunk 2]
```diff
@@ -151,42 +151,50 @@ api-config {
     scopes {
         "read:organisations-details-ho-ssp" {
             fields = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"]
+            endpoints = []
```
Comment (owen-opencast) [inline conf/application.conf:154]: Don't think we need this, can be omitted

**Review by owen-opencast — CHANGES_REQUESTED** [review]

File: app/uk/gov/hmrc/organisationsdetailsapi/services/NumberOfEmployeesService.scala [hunk 3]
```diff
@@ -38,10 +38,13 @@ class NumberOfEmployeesService @Inject()(
 
   def get(matchId: UUID, employeeCountRequest: NumberOfEmployeesRequest, scopes: Iterable[String])
          (implicit hc: HeaderCarrier, request: RequestHeader, ec: ExecutionContext): Future[Option[Seq[NumberOfEmployeesResponse]]] = {
+
+    val endpointName = "number-of-employees";
```
Comment (ajchilts) [inline app/uk/gov/hmrc/organisationsdetailsapi/services/NumberOfEmployeesService.scala:42]: No semi colon needed :-)

**Review by ajchilts — CHANGES_REQUESTED** [review]

File: app/uk/gov/hmrc/organisationsdetailsapi/services/NumberOfEmployeesService.scala (hunk 3 above)
Comment (parkeradam) [reply app/uk/gov/hmrc/organisationsdetailsapi/services/NumberOfEmployeesService.scala:42]: Oops. find my language background.... xD

**Review by ajchilts — APPROVED** [review]

**Review by owen-opencast — APPROVED** [review]

**Pull request merged**